### PR TITLE
Export: include alerts, thumbnails, usage stats, and short urls

### PR DESCRIPTION
**Pull request by ryantxu**
This PR expands the export capabilities to include a few more tables

### CODE REVIEW

File: pkg/services/export/export_auth.go [hunk 1]
```diff
@@ -13,58 +15,116 @@ func dumpAuthTables(helper *commitHelper, job *gitExportJob) error {
...
-			}
-			if len(raw) < 1 {
-				continue // don't write empty files
+				return err
 			}
-			frame, err := queryResultToDataFrame(raw, frameOpts{
-				skip: []string{"org_id", "version", "help_flags1", "theme"},
-			})
+
+			frame, err := sqlutil.FrameFromRows(rows.Rows, -1, auth.converters...)
```
Comment (ryantxu) [inline pkg/services/export/export_auth.go:103]: @ArturWierzbicki -- in a previous PR, you asked for tests on how the `queryResultToDataFrame` works.  In this PR, I am removing the whole `frame_helper.go` and using the version from our SDK https://github.com/grafana/grafana-plugin-sdk-go/blob/main/data/sqlutil/sql.go#L22

File: pkg/services/export/export_auth.go [hunk 2]
```diff
@@ -13,58 +16,119 @@ func dumpAuthTables(helper *commitHelper, job *gitExportJob) error {
...
+				table: "user",
+				sql: `
+					SELECT user.*, org_user.role 
+					  FROM user 
+					  JOIN org_user ON user.id = org_user.user_id
+					 WHERE org_user.org_id =` + strconv.FormatInt(helper.orgID, 10),
+				converters: []sqlutil.Converter{{Dynamic: true}},
+				drop: []string{
+					"id", "version",
+					"password", // UMMMMM... for now
```
Comment (nmarrs) [inline pkg/services/export/export_auth.go:37]: uh

**Review by nmarrs — APPROVED** [review]
Tested locally (paired) and functionality looks good to me, overall go code looks reasonable to me (though limited knowledge base)

File: pkg/services/export/export_auth.go (hunk 2 above)
Comment (ryantxu) [reply pkg/services/export/export_auth.go:37]: don't want to export passwords right now... given that we can not really import that is good enough :)

**Pull request merged**